### ISSUE TITLE
Multi-window use

### ISSUE THREAD

**Issue by kleinerpirat** [fonol/anki-search-inside-add-card#90]
**Warning: If you're using the addon's note system, I don't recommend doing this, since I haven't tested it. Read the author's comment below my post.** 

It's possible to use this Addon with multiple windows.
Sharing it here in the hopes of helping someone who might want this.

**My use-case:**
Usually I create my notes by having a slim add-window on the left side of my primary monitor, which takes up about a quarter of the screen's width, while browsing through my sources on Chrome or an external PDF-Reader on the remaining space of that same monitor. My second monitor is used for the card-browser (among other things).
I found this to be the most efficient way of creating cards for me (I tried using your inbuilt PDF-Reader and i loved the incremental reading aspect, but i like having more flexibility in my workflow + the PDF's don't look sharp enough for my taste).

I really like your addon's search-on-type feature, but find it most useful when it is always visible (e.g. on a second/third monitor). With my slim add-window I would have to toggle to see the search results. So I searched and found [Opening the same window multiple time](https://ankiweb.net/shared/info/354407385) by Arthur Milchior.

**TL;DR:**
- Install [Opening the same window multiple time](https://ankiweb.net/shared/info/354407385)
- Step 2: open "primary" add-window
- Step 3: open "secondary" add-window
- done!

Now everything you type in the primary add-window is synchronously searched in the secondary window. 

![multiwindow](https://user-images.githubusercontent.com/62722460/80961376-22ed8f80-8e0b-11ea-9e56-41329352940c.gif)
**Info:** This was recorded on an old version of the Addon. Now it works without closing and reopening the second window.

**Small tweak**
I recommend changing "Left side width" in the addon settings to 0%, so the secondary window is purely for showing results. I keep that window on another monitor, that way I always have the search results, even with my slim-add-window-workflow.

**Sample 2-monitor-setup**
Monitor 1
<img width="1280" alt="left monitor" src="https://user-images.githubusercontent.com/62722460/80975477-70292b80-8e22-11ea-9041-6fced3f31e7c.png">
Monitor 2
<img width="960" alt="right monitor" src="https://user-images.githubusercontent.com/62722460/80975541-7ddeb100-8e22-11ea-8578-f80ee15d76f0.png">

**Comment by fonol** [comment]
Hi, that is really cool. thanks for the write-up.
I don't have a clue why it works after reopening the window a second time, 
when I'm bored sometime, I might try to find out why!
For now, happy that it works for you.
Some word of advice: Don't try to do stuff that persists (e.g. stuff around the add-on's notes) while having two windows. It might really mess things up, as lots of state in the add-on is basically in the UI. But I guess you just want the search feature anyway. 
On a sidenote: May I ask for the resolution of your monitor, because you wrote of the pdfs being blurry?

**Comment by kleinerpirat** [comment]
Thanks for the heads-up!
Correct, I only intend to use the search feature, so I think I'll be good, but if I notice any irregularities I'll post them here.

My monitor's resolution is 2560x1440. So your PDF-reader probably displays the content at HD resolution, right? So that could be why it looked blurry to me.

**Comment by fonol** [comment]
I think the problem is me not getting how the scaling works on pdf.js.
I guess with the right setting, it would look perfectly sharp. Oh well.
